packages/nimble-components/src/anchor-menu-item/styles.ts

```diff
@@ -0,0 +1,79 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+
+import {
+    bodyDisabledFontColor,
+    bodyFont,
+    bodyFontColor,
+    borderHoverColor,
+    controlHeight,
+    fillHoverColor,
+    fillSelectedColor,
+    iconSize
+} from '../theme-provider/design-tokens';
+
+export const styles = css`
+    ${display('grid')}
+
+    :host {
+        font: ${bodyFont};
+    }
+
+    :host(:focus-within:not(:active)) {
+        outline: 2px solid ${borderHoverColor};
+        outline-offset: -2px;
+    }
+    :host(:hover) {
+        background: ${fillHoverColor};
+    }
+    :host(:active) {
+        background: ${fillSelectedColor};
+    }
+    :host([disabled]) {
+        background: transparent;
+    }
+    a {
+        display: grid;
```

Comment:
It looks like there is a lot of duplicated styling between this new component and the existing menu-item styles. I'm curious if you considered single-sourcing the styles or thought about what it might look like. Some of the selectors are the same between the two components, but some have different selectors, which is different than when we've shared styling in the past.

Reply:
I did start out trying to share it, but decided that they diverged enough that it wasn't worth trying to share.